src/game/Rockstar.svelte

```diff
@@ -20,6 +34,9 @@
     onDestroy(clearAllTimeouts);
 
     $: if (rockstar) {
+        if (difficulty === 'hard') {
+            button = shuffle(button);
```

Comment:
Je pense que tu peux utiliser le shuffle vanilla (normalement pas besoin de réaffectation, mais à tester):
```suggestion
            button.shuffle();
```